AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -92,6 +93,7 @@ public CallingContext(final Context applicationContext, final Callable<String> t
         remoteParticipantsMap = new HashMap<>();
         displayedRemoteParticipants = new ArrayList<>();
         displayedParticipantsLiveData = new MutableLiveData<>();
+        remoteParticipantUpdate = new MutableLiveData<>();
```

Comment:
For now, I think we can continue to use the existing `displayedParticipantsLiveData`.

We can then have a separate optimization in the future to minimize view updates when the state of the participant has not changed.

Reply:
On the CallingActivity side there's some logic to add some delay between view updates when there are rapid events. There's probably more complex logic we could introduce to limit the events from firing, but I think we can handle that separately unless it causes a major issue.

Reply:
I changed it to use `displayedParticipantsLiveData`